assets/js/ViewerApp.tsx

```diff
@@ -170,6 +180,17 @@ class ViewerApp extends React.Component<
     }
   }
 
+  setSignGroup(ts: number, signGroup: SignGroup): void {
+    const { signGroupsChannel: channel } = this.state;
+
+    if (channel) {
```

Comment:
It's not present when the component's constructor is invoked and initializes the state. However, the channels are joined upon `componentDidMount`. So, I think it's correct for its type to be `Channel | null`, and to have to type-guard the null here, though in practice it should always be present by the time this function is invoked.

Reply:
If it's something we expect should always be present (or else there's a bug in the code), maybe we can instead do `channel!.push(...)`? Then this would throw an exception, which would show up in Sentry, and we'd know it happened.

Reply:
Interesting, yeah. Good idea.

Reply:
Hm, actually that runs afoul of a typescript lint we have configured, and I'm reluctant to disable that lint generally. However, I do like the idea of hearing in Sentry if there's an issue, so I'll use our JS Sentry package in an `else` block to send a message if it's not present for some reason.

Reply:
Equivalently to the `!` method, we could just `throw new Error("...")` and Sentry should pick that up without having to explicitly call it.